src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -233,28 +233,33 @@ class DefaultSource
           new StringDataPointsRelationV1(config)(sqlContext)
         case _ => sys.error(s"Resource type $resourceType does not support save()")
       }
+      val batchSize = relation match {
+        case n: NumericDataPointsRelationV1 => Constants.CreateDataPointsLimit
```

Comment:
I think this should also apply to string data points, does it?

Reply:
It should, I just missed it. I'll implement it, thanks for pointing it out.